src/main/index.js

```diff
@@ -47,16 +47,12 @@ function createMainWindow() {
 // quit application when all windows are closed
 app.on('window-all-closed', () => {
   // on macOS it is common for applications to stay open until the user explicitly quits
-  if (process.platform !== 'darwin') {
-    app.quit()
-  }
+  process.platform === 'darwin' || app.quit()
```

Comment:
This looks confusing :upside_down_face:

Reply:
maybe something more easy to read like:  `(process.platform !== 'darwin') && app.quit()`

Reply:
Anyway this changes aren't really necessary, try the components folder...